niri-config/src/lib.rs

```diff
@@ -389,6 +389,8 @@ pub struct Layout {
     pub default_column_width: Option<DefaultColumnWidth>,
     #[knuffel(child, unwrap(argument), default)]
     pub center_focused_column: CenterFocusedColumn,
+    #[knuffel(child, unwrap(argument), default = false)]
```

Comment:
```suggestion
    #[knuffel(child)]
```
Bool flags can work like this, then if they are present it counts as true, and if absent, it counts as false.